include/sys/kasan.h

```diff
@@ -19,6 +19,7 @@
 #define KASAN_CODE_POOL_FREED 0xFD
 #define KASAN_CODE_KMALLOC_OVERFLOW 0xFE
 #define KASAN_CODE_KMALLOC_FREED 0xFF
+#define KASAN_CODE_NEW 0xDF
```

Comment:
Invalid codes must have values in the range `0xf1-0xff`, see comment above `kasan_mark()`.

Reply:
done